news/2024/2024-05-01-the-nat-gazette-june-2024.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "The NAT Gazette: June 2024"
+date: 2024-05-01 13:00:00 +0000
+---
+
+The NAT Gazette returns with a big bang and a bigger list of changes to cover!
+
+![](/wiki/shared/news/2024-05-01-the-nat-gazette-june-2024/nat-gazette-june-2024.jpg)
+
+The NAT isn't dead, nor is the long awaited batch of updates that have slowly been creeping in ever since the last issue. With a bigger emphasis placed on transparency and easing communication between the Beatmap Nominators and the general playerbase, we'd like to shift gears and focus on how users receive this full suite of updates.
+
+While there's a lot to go through, our resident bird connoisseur [pishifat](https://osu.ppy.sh/users/3178418) has prepared us a changelog in the form of a video! We encourage giving it a look to quickly be up to speed with all the changes to come.
+
+<div align="center">
```

Comment:
makes bottom margin the same as other paragraphs
(this should be in the news styling criteria templates shouldn't it)

![image](https://github.com/ppy/osu-wiki/assets/36758269/7b10a48e-af1c-41ef-a681-384360636fdc)

```suggestion
<div align="center" class="osu-md__paragraph">
```